terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/src/test/java/org/terasoluna/gfw/web/pagination/PaginationTagTest.java

```diff
@@ -82,7 +86,27 @@ public void tearDown() throws Exception {
 
     @Test
     public void testRelease01() throws Exception {
+
         tag.release();
+
+        assertNull(getField(tag, "page"));
```

Comment:
- [ ] I think you may not use static import `ReflectionTestUtils.getField`.
because it's not common to recall using `ReflectionTestUtils` from `getField`.

Reply:
fix on baf904b5bd1a97115ed1d04ec6bf42e14f477fff